plugins/inputs/modbus/modbus.go

```diff
@@ -246,9 +250,16 @@ func (m *Modbus) initClient() error {
 		if err != nil {
 			return err
 		}
-		handler := mb.NewTCPClientHandler(host + ":" + port)
-		handler.Timeout = time.Duration(m.Timeout)
-		m.handler = handler
+		switch m.TransmissionMode {
+		case "RTU":
+			handler := mb.NewRTUOverTCPClientHandler(host + ":" + port)
+			handler.Timeout = time.Duration(m.Timeout)
+			m.handler = handler
+		default:
+			handler := mb.NewTCPClientHandler(host + ":" + port)
+			handler.Timeout = time.Duration(m.Timeout)
+			m.handler = handler
+		}
```

Comment:
Please use `"RTUoverTCP"` and also implement `"ASCIIoverTCP"` in the same PR.

Reply:
The different transmission modes now are: `RTUoverTCP`, `ASCIIoverTCP`, `TCP` (default) for TCP, `RTU` and `ASCII`